### PR TITLE
Fix broken links in Getting Started guide

### PR DESCRIPTION
**Pull request by boczkowska**
## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

**Comment by amitkrout** [comment]
```
✗  Unable to connect to OpenShift cluster, is it down?
```
/test integration

File: docs/getting-started.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ This document walks you through the development and deployment of a Node.js appl
 == Developing and deploying an application to an OpenShift cluster
 To develop and deploy a Node.js application to an OpenShift cluster using `odo`:
 
-. link:/docs/installation.md[Install the `odo` binary].
+. link:https://openshiftdo.org/installation/[Install the `odo` binary].
```
Comment (surajnarwade) [inline docs/getting-started.adoc]: we can simply points to `installation.adoc`

Comment (boczkowska) [reply docs/getting-started.adoc]: @surajnarwade changed

Comment (surajnarwade) [reply docs/getting-started.adoc]: :+1:

**Pull request merged**